sirepo/sim_data/flash.py

```diff
@@ -26,8 +28,21 @@ def fixup_old_data(cls, data):
             )
 
     @classmethod
-    def proprietary_lib_file_basename(cls, data):
-        return '{}.zip'.format(sirepo.util.secure_filename(data.models.simulation.flashType))
+    def flash_exe_name(cls, data):
+        return '{}-{}'.format(
+            cls._FLASH_EXE_PREFIX,
+            data.models.simulation.flashType,
+        )
+
+    @classmethod
+    def flash_setup_units_path(cls, data):
+        return pkio.py_path(
+            # TODO(e-carlin): talk with rn about sharing this path better with installer
+            '/home/vagrant/.local/share/flash4/{}-{}'.format(
```

Comment:
@robnagler should this be shared better?

Reply:
Yes, this should be configured, not hardwired like this. It should be relative to the exe, which we can find. We know it will be called `share`. flash_exe_name should probably be flash_exe_path which returns a py.path that can be manipulated to get to `share/flash4`.

Reply:
https://github.com/radiasoft/sirepo/pull/2490/commits/00903ca4af2b437c4de426a53cfab1f8b79638eb